mender/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "3.7.4"
 description: Mender is a robust and secure way to update all your software and deploy your IoT devices at scale with support for customization
 name: mender
-version: 5.6.2
+version: 5.6.3
```

I would argue that since you're adding features to the helm chart, this should be a minor bump.
```suggestion
version: 5.7.0
```

---

Right!